### PR TITLE
common: log: Add debug log relative shell commands

### PR DESCRIPTION
**Pull request by MouchenHung-QUANTA**
Summary:
- Able to switch log debug status while runtime with shell command in BIC console.

Test Plan:
- Build Code: PASS
- Check log in BIC console: PASS

Log:
- BIC console:
  ```
  uart:~$ platform log list_all
  --------------------------
  [0 ] DEBUG_I2C      : x
  [1 ] DEBUG_PECI     : x
  [2 ] DEBUG_KCS      : x
  [3 ] DEBUG_IPMB     : x
  [4 ] DEBUG_IPMI     : x
  [5 ] DEBUG_SENSOR   : x
  [6 ] DEBUG_USB      : x
  --------------------------
  uart:~$ platform log control 6 1
  Log 6 status set 1 success!
  uart:~$ platform log control 5 1
  Log 5 status set 1 success!
  uart:~$ platform log control 4 1
  Log 4 status set 1 success!
  uart:~$ platform log list_all
  --------------------------
  [0 ] DEBUG_I2C      : x
  [1 ] DEBUG_PECI     : x
  [2 ] DEBUG_KCS      : x
  [3 ] DEBUG_IPMB     : x
  [4 ] DEBUG_IPMI     : o
  [5 ] DEBUG_SENSOR   : o
  [6 ] DEBUG_USB      : o
  --------------------------
  uart:~$ platform log control 6 0
  Log 6 status set 0 success!
  uart:~$ platform log list_all
  --------------------------
  [0 ] DEBUG_I2C      : x
  [1 ] DEBUG_PECI     : x
  [2 ] DEBUG_KCS      : x
  [3 ] DEBUG_IPMB     : x
  [4 ] DEBUG_IPMI     : o
  [5 ] DEBUG_SENSOR   : o
  [6 ] DEBUG_USB      : x
  --------------------------
  uart:~$ platform log halt
  All log has been halt!
  uart:~$ platform log list_all
  --------------------------
  [0 ] DEBUG_I2C      : x
  [1 ] DEBUG_PECI     : x
  [2 ] DEBUG_KCS      : x
  [3 ] DEBUG_IPMB     : x
  [4 ] DEBUG_IPMI     : x
  [5 ] DEBUG_SENSOR   : x
  [6 ] DEBUG_USB      : x
  --------------------------
  ```

### CODE REVIEW

File: common/log/log_util.c [hunk 1]
```diff
@@ -0,0 +1,35 @@
+#include <zephyr.h>
+#include <sys/printk.h>
+#include <stdio.h>
+#include <string.h>
+#include <stdlib.h>
+
+#include "log_util.h"
+
+#define log_name_to_num(x) #x,
+const char *const log_name[] = { log_name_to_num(DEBUG_I2C) log_name_to_num(DEBUG_PECI)
```
Comment (garnermic) [inline common/log/log_util.c:10]: Please reformat or run linter to make this list easier to read.

**Review by garnermic — CHANGES_REQUESTED** [review]
Minor formatting change requested.

File: common/log/log_util.c (hunk 1 above)
Comment (MouchenHung-QUANTA) [reply common/log/log_util.c]: Thank you for your reply, new commit has modified this format issue.

**Comment by GoldenBug** [comment]
A separate logging facility that should be built upon has been added to the shell using Zephyrs built in logging capabiltiies.

**Comment by GoldenBug** [comment]
A separate logging facility has been added to the shell

**Pull request closed without merging**